index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
+console.log("%cQuestion 1", "color: red")
+let counter = 1
+counter = 2
+console.log(counter)
 // // => 2
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red")
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
+// *** after testing, comment the line causing the error back in
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red")
+const name = "James"
```

Comment:
Nice! The key here was using the keyword `const` because it doesn't allow us to reassign our variable